json-guides/circuit-breaker.json

```diff
@@ -159,11 +159,12 @@
           "browserContent": "/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/check-balance-fail-with-open-circuit.html",
           "description":
           [
-            "The CircuitBreaker policy opens the circuit when sufficient calls to the microservice fail. The policy determines the failure threshold by looking at the latest requests to the service, also known as a <b>rolling window</b>. You can modify this failure threshold by using these two parameters:<br/>",
-            "<ul><li><b>requestVolumeThreshold:</b> The size of the rolling window that is used to determine the failure threshold. The default is <code>20</code>.",
-            "<li><b>failureRatio:</b> The minimum failure ratio in the rolling window that triggers the circuit breaker. The default is <code>0.5</code>.",
+            "The CircuitBreaker policy opens the circuit when the failure threshold is met. When the circuit is open, calls to the failing service will fail immediately. The policy determines the failure threshold by monitoring a specified number of the most recent consecutive requests made to the service, also known as a <b>rolling window</b>. You can modify the failure threshold by using these two parameters:<br/>",
+            "<ul><li><b>requestVolumeThreshold:</b> The minimum number of requests that must be made before determining if the circuit should trip.  It indicates the size of the rolling window. The default is <code>20</code> requests.",
```

Comment:
Possible suggestion: `This also indicates the size of ...`

Reply:
Ok....I'll put it in and see what ID thinks.